client/src/components/footer/index.js

```diff
@@ -5,18 +5,17 @@ import {Notifications} from '../notifications';
 
 const style = {
   footer: {
-    position: 'absolute',
+    zIndex: 9999,
+    position: 'fixed',
     bottom: '0',
-    width: '100%',
+    width: '90%',
```

Comment:
I'd either use bootstrap styles to scale it properly (e.g. `col-xs-12`) or change styles to use flexbox.